src/components/toc/_macro.njk

```diff
@@ -0,0 +1,20 @@
+{% macro onsToc(params) %}
```

Comment:
Should these terms and conditions files be part of this PR?

Reply:
I branched off the toc branch as it had the lists update in it. When I update the toc branch i'll merge it in to this.